rdrf/report/schema.py

```diff
@@ -104,6 +104,12 @@ def cde_resolver(cdes, _info, cde_model):
                     value = cde_value["value"]
                     if datatype == 'lookup':
                         return get_widget_class(cde_model.widget_name).denormalized_value(value)
+                    elif cde_model.pv_group:
+                        cde_values_dict = cde_model.pv_group.cde_values_dict
```

Comment:
Possible to prefetch / cache the pv_group?

Reply:
The parse_utils.prefetch_form_data method (where the form, section, cdes for reporting are sourced from), is cached and also preselects the pv_group. 
https://github.com/eresearchqut/trrf/blob/next_release/rdrf/rdrf/forms/dsl/parse_utils.py#L53

That should cover it?